### ISSUE TITLE
Are there any plans to make a flatpak version?

### ISSUE THREAD

**Issue by barfin** [FWGS/xash3d-fwgs#1259]

**Comment by a1batross** [comment]
flatpak for 64-bit is in the works

but I don't think we need it

I built 32-bit flatpak just for Steam Deck

**Comment by barfin** [comment]
> flatpak for 64-bit is in the works
> 
> but I don't think we need it
> 
> I built 32-bit flatpak just for Steam Deck, so you could install half-Life from Steam and game files would automatically be fetched from this installation

yeah, I don't think half-life or a half-life mod will ever use more than 4gb of ram, 32-bit should be good enough, will xash3d get uploaded to flathub when the right time comes or is that something that's currently (or in the near future) impossible?

**Comment by a1batross** [comment]
Yeah, when we release it.

Or I may host my own repository for test builds

пт, 10 мар. 2023 г., 10:33 shortcircuit ***@***.***>:
